### DISCUSSION TITLE
是否支持rgcn算法，即多种类型的边

### DISCUSSION THREAD

**Discussion by bookug** [alibaba/GraphScope#1406] [Q&A] [answered]
graphkearn仓库中有这个算法，但scope仓库反而没看到？

**Comment by bookug** [comment, marked as answer]
graphscope是用pip安装的，在site packages中看到有gcn，但没有rgcn

**Reply by sighingnow** [reply]
GraphScope tracks the `graphscope` branch of graph-learn's repo.

>  在site packages中看到有gcn，但没有rgcn

`rgcn` algorithm is not included in the `graphscope` branch. Such algorithms might be backported to the `graphscope` branch but we don't have such ETA yet.

You could implement such algorithms upon the APIs under `graphscope.learning...`.

**Comment by bookug** [comment]
有没有一个详细的安装文档，特别是源码安装文档，图学习部分

**Reply by sighingnow** [reply]
You could refer to https://graphscope.io/docs/zh/developer_guide.html, and free to ask if you meets further questions.

**Comment by sighingnow** [comment]
Hi @bookug,

I have moved the issue here to makes the disscussion clearer.
